test/unit/test_TheorySimplifications.cc

```diff
@@ -131,14 +128,13 @@ TEST_F(RetrieveSubstitutionTest, test_NestedSubstitution) {
     vec<PtAsgn> facts;
     facts.push(PtAsgn{eq, l_True});
     facts.push(PtAsgn{eq2, l_True});
-    MapWithKeys<PTRef,PtAsgn,PTRefHash> subst;
-    logic.retrieveSubstitutions(facts, subst);
-    ASSERT_TRUE(subst.has(z));
-    ASSERT_TRUE(subst.has(y));
+    auto subst = logic.retrieveSubstitutions(facts);
+    ASSERT_TRUE(subst.second.has(z));
+    ASSERT_TRUE(subst.second.has(y));
     PtAsgn afx = PtAsgn{fx, l_True};
     PtAsgn afy = PtAsgn{fy, l_True};
-    EXPECT_EQ(subst[z], afy);
-    EXPECT_EQ(subst[y], afx);
+    EXPECT_EQ(subst.second[z], afy.tr);
+    EXPECT_EQ(subst.second[y], afx.tr);
```

Comment:
```suggestion
    EXPECT_EQ(subst.second[z], fy);
    EXPECT_EQ(subst.second[y], fx);
```
And `afx, afy` can be removed.

Reply:
Fixed,